app/com/gitpitch/services/ImageService.java

```diff
@@ -66,8 +67,8 @@ public String buildBackgroundOffline(String md) {
             String fragUrl = frag.substring(0, frag.indexOf("\""));
             String imageName = FilenameUtils.getName(fragUrl);
             String imageUrl = IMG_OFFLINE_DIR + imageName;
-            return buildBackground(null, imageUrl);
 
```

Comment:
@jroper a number of changes have been pushed live to address the following:

1. Your PR with support for `background-size` has been merged and is now live.
2. Issue #9 related to offline background images has been resolved and is a fix is now live.
3. A new caching policy is in place for print and offline export bundles. Now you can use feature branches and experience just 20 second caching, so good for dev cycle. Master branch bundles continue to be cached for 20 minutes.

[Here is a link to your slideshow](https://gitpitch.com/jroper/microservices-architecture/8b7d3f74725cd1219dffa14432af6c871e33223a) where you left off working. Click it to see if these fixes resolve all issues.